### ISSUE TITLE
Pretrained INaturalist models llink is not working

### ISSUE THREAD

**Issue by alishsuper** [Megvii-Nanjing/BBN#1]
I've tried to download the pretrarined INaturalist 2018 model but the link https://pan.baidu.com/error/404.html is not working anymore. Could you please share with me working link?

Thanks in advance!

**Comment by ZhouBoyan** [comment]
I can open this link (https://pan.baidu.com/s/1olDppTptZ5HYWsgQsMCPLQ) normally, I wonder if there is something wrong with your internet.

**Comment by AlessandroSaviolo** [comment]
Did you train your model from scratch or using pretrained weights? If the latter, which one did you use?
Thanks

**Comment by ZhouBoyan** [comment]
All of the models are trained from scratch.

**Comment by alishsuper** [comment]
@ZhouBoyan what the best accuracy you got?

**Comment by alishsuper** [comment]
> I can open this link (https://pan.baidu.com/s/1olDppTptZ5HYWsgQsMCPLQ) normally, I wonder if there is something wrong with your internet.

I am in Taiwan now and I was trying to download weights from here. Baidu doesn't work here...

**Comment by alishsuper** [comment]
Could you kindly share with me the pretrained weights with the best results for INaaturalist 2019? I would be very grateful to you!

**Comment by ZhouBoyan** [comment]
> > I can open this link (https://pan.baidu.com/s/1olDppTptZ5HYWsgQsMCPLQ) normally, I wonder if there is something wrong with your internet.
> 
> I am in Taiwan now and I was trying to download weights from here. Baidu doesn't work here...

We have updated our README with the link of google drive to download the weights. You could try again with that link.

**Comment by alishsuper** [comment]
> > > I can open this link (https://pan.baidu.com/s/1olDppTptZ5HYWsgQsMCPLQ) normally, I wonder if there is something wrong with your internet.
> > 
> > 
> > I am in Taiwan now and I was trying to download weights from here. Baidu doesn't work here...
> 
> We have updated our README with the link of google drive to download the weights. You could try again with that link.

Thank you very much for your reply! I'm really appreciate this. You shared with me weights for iNaturalist 2018, but I need weights for iNaturalist 2019. Could you please share with me?

**Comment by zilunzhang** [comment]
> > > > I can open this link (https://pan.baidu.com/s/1olDppTptZ5HYWsgQsMCPLQ) normally, I wonder if there is something wrong with your internet.
> > > 
> > > 
> > > I am in Taiwan now and I was trying to download weights from here. Baidu doesn't work here...
> > 
> > 
> > We have updated our README with the link of google drive to download the weights. You could try again with that link.
> 
> Thank you very much for your reply! I'm really appreciate this. You shared with me weights for iNaturalist 2018, but I need weights for iNaturalist 2019. Could you please share with me?

I think the paper only uses iNaturalist 2017 and 2018 as benchmark dataset, they may not have weights for iNaturalist 2019.
